src/nimlsp.nim

```diff
@@ -166,7 +167,7 @@ if not existsFile(nimpath / "config/nim.cfg"):
   stderr.write "Unable to find \"config/nim.cfg\" in \"" & nimpath & "\". " &
     "Supply the Nim project folder by adding it as an argument.\n"
   quit 1
-
+{.effects.}
```

Comment:
What is this for?

Reply:
check effects, but dont see it print to output.

Reply:
But why?

Reply:
never mind, no need this now.